# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/svg/svg-integration/clip-path/reftest.list
# Commit: 8d13c0ab66f0
# Full Hash: 8d13c0ab66f0ac137b1461d3451ecb9361ec2d5e
# Author: Jamie Nicol <jnicol@mozilla.com>
# Date: 2023-03-22 21:15:54
# Regressor Bug: 1823411
# File Overlap Count: 1
# Description:
#   Bug 1823411 - Give all webrender's shader varyings an explicit precision. r=gfx-reviewers,lsalzman
#   
#   Mali profiling tools have shown we are frequently fragment bound, in
#   particular due to varying interpolation. To help mitigate this, we
#   should use mediump where possible. Currently most of our varyings are
# ==============================================================================

diff -r 3919fbd972a6 -r 8d13c0ab66f0 layout/reftests/svg/svg-integration/clip-path/reftest.list
--- a/layout/reftests/svg/svg-integration/clip-path/reftest.list	Tue Mar 21 19:04:22 2023 +0000
+++ b/layout/reftests/svg/svg-integration/clip-path/reftest.list	Tue Mar 21 19:05:19 2023 +0000
@@ -14,7 +14,7 @@
 == clip-path-polygon-010.html clip-path-stripes-001-ref.html
 == clip-path-polygon-011.html clip-path-stripes-001-ref.html
 == clip-path-polygon-012.html clip-path-stripes-001-ref.html
-fuzzy(0-1,0-20) == clip-path-polygon-013.html clip-path-stripes-003-ref.html
+fuzzy(0-1,0-20) fuzzy-if(Android&&device&&!swgl,255-255,40-48) == clip-path-polygon-013.html clip-path-stripes-003-ref.html
 
 fuzzy-if(!useDrawSnapshot,35-70,699-800) == clip-path-circle-001.html clip-path-circle-001-ref.html
 fuzzy-if(!useDrawSnapshot,35-70,699-800) == clip-path-circle-002.html clip-path-circle-001-ref.html
